src/detect-icode.c

```diff
@@ -311,7 +311,6 @@ static int DetectICodeParseTest08(void)
     DetectU8Data *icd = DetectU8Parse("> 8 <> 20");
     FAIL_IF_NOT_NULL(icd);
 
-    DetectICodeFree(NULL, icd);
```

Comment:
should probably annotate the func with nonnull attribute for that ptr

Reply:
Is it done in some other function ? (`git grep nonnull` does not show anything like that)

Reply:
iirc it is `__attribute__((nonnull)))`